Improve instructions for updating allowlist

FrNecas opened the pull request:
As discussed today, lets make it a bit clearer in the README what the correct format is. @mfocko is there anything else you'd like to add?
---
None

TomasTomecek reviewed (approved):
definitely more clear

mfocko reviewed (approved):
`waiting` can be also interactive, since you can pick pending namespace and approve it from there, otherwise LGTM

FrNecas commented:
Thanks for the hint with interactive `waiting`. I adjusted the prompt a little bit (the message that was there seemed like a `yes/no` question when in fact I was to specify an integer) and added a mention of this to the README.
